### PR TITLE
Update node-density-heavy tests to use kube burner ocp wrapper.

### PR DESCRIPTION
**Pull request by svetsa-rh**

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-namespaced-3nodes

File: ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh [hunk 1]
```diff
@@ -18,7 +18,10 @@ GSHEET_KEY_LOCATION="/ga-gsheet/gcp-sa-account"
 export GSHEET_KEY_LOCATION
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
-pushd e2e-benchmarking/workloads/kube-burner
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+#export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE "
+#export NAMESPACED_ITERATIONS=${NAMESPACED_ITERATIONS:-false}
+#export EXTRA_FLAGS=${EXTRA_FLAGS:-}
```
Comment (krishvoor) [inline ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:24]: These variables can be deleted, as they are referenced in [openshift-qe-node-density-heavy-ref.yaml](https://github.com/openshift/release/pull/41745/files#diff-fd8637f16f5b639fef15ec39022b495a5a84001e7dbf87407f22cceb3dbbbdc5)

File: ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh [hunk 2]
```diff
@@ -18,7 +18,10 @@ GSHEET_KEY_LOCATION="/ga-gsheet/gcp-sa-account"
...
 export WORKLOAD=node-density-heavy
 
 export CLEANUP_WHEN_FINISH=true
```
Comment (krishvoor) [inline ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:27]: We can capture these configurable variables in [openshift-qe-node-density-heavy-ref.yaml](https://github.com/openshift/release/pull/41745/files#diff-fd8637f16f5b639fef15ec39022b495a5a84001e7dbf87407f22cceb3dbbbdc5) and override
as required in [openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml](https://github.com/openshift/release/pull/41745/files#diff-99f37dbb0b40d6b2cafd605b48b7b13ab9194dedfc1b6986543d7a669cbf0e54)

Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:27]: the cleanup variable for kube-burner ocp way of running is GC, might be nice to add this as an envionrment variable

Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:27]: Agreed. We can open a separate PR to tackle that and apply for all relevant workloads.

Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:27]: May be Sahil can tackle that as part of having common code across various kube burner workloads.

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-namespaced-3nodes

File: ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh (hunk 1 above)
Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:24]: I plan to remove them once tests pass Harsha. Not sure why I added PODS_PER_NODE to EXTRA_FLAGS before. So, wanted to make sure tests run successfully before I delete it.

File: ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh (hunk 2 above)
Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh:27]: Right now, we have these for all workloads and I agree that they have to be moved to upper level. 
I think they should be tackled as part of https://issues.redhat.com/browse/OCPQE-15384

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-node-density-heavy-gcp

File: ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml [hunk 3]
```diff
@@ -21,6 +21,14 @@ ref:
     default: "230"
```
Comment (afcollins) [inline ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml:21]: suggest making default here 100. 230 is too intense.

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-namespaced-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-namespaced-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-namespaced-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-namespaced-3nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-node-density-heavy-gcp

**Comment by svetsa-rh** [comment]
@paigerube14 @jtaleric Could you ptal and approve? 
The failures are unrelated to the tests and I pushed another commit to update OUTPUT_FILE.

**Comment by svetsa-rh** [comment]
@afcollins Could you ptal? I updated it based on our conversation. Thanks!

**Review by afcollins — APPROVED** [review]
LGTM

**Pull request merged**